### ISSUE TITLE
qwen2-1.5b + swift , loss nan

### ISSUE THREAD

**Issue by XiaoMaGe-hero** [modelscope/swift#1175]
使用swift lora  qwen2-1.5b or qwen2-1.5b-instruct, 开启训练时 loss nan。替换模型为 qwen2-0.5b 正常

**Comment by XiaoMaGe-hero** [comment]
1. 使用本地下载的qwen2-1.5B模型能够正常推理 
2. lora  qwen2_1_5b 过程中遇到loss nan  ，当前版本是否支持 lora qwen2_1_5b ？ 
代码：
import os
import sys
import torch
torch.cuda.empty_cache()

os.environ["CUDA_VISIBLE_DEVICES"] = "0"

from swift.llm import (
    DatasetName,
    InferArguments,
    ModelType,
    SftArguments,
    infer_main,
    sft_main,
    app_ui_main,
    merge_lora,
)

model_path = "xxxxx/Qwen2-1.5B"

model_type = ModelType.qwen2_0_5b_instruct
model_type = ModelType.qwen2_1_5b_instruct
sft_args = SftArguments(
    model_type=model_type,
    model_id_or_path="xxxxx/Qwen2-1.5B",
    # train_dataset_sample=800, 
    dataset_test_ratio=0.1,
    val_dataset_sample=-1,
    save_total_limit=4,
    lora_rank=8,
    lora_alpha=32,
    lora_dropout_p=0.05,
    # dataset=[DatasetName.blossom_math_zh],
    dataset="xxxxxsamples_v2.jsonl",
    output_dir="xxxxx/Qwen2_1_5b_instruct",
    num_train_epochs=5,
    eval_steps=2,
    #use_galore=False,
)

result = sft_main(sft_args)
best_model_checkpoint = result["best_model_checkpoint"]
print(f"best_model_checkpoint: {best_model_checkpoint}")
torch.cuda.empty_cache()

infer_args = InferArguments(
    ckpt_dir=best_model_checkpoint, load_dataset_config=True, val_dataset_sample=100
)
merge_lora(infer_args, device_map="cuda:0")
result = infer_main(infer_args)
torch.cuda.empty_cache()

**Comment by Jintao-Huang** [comment]
dtype设置为bf16呢

**Comment by XiaoMaGe-hero** [comment]
当前设备不支持bf16，使用 fp32后loss正常了
